src/functions-reference/real-valued_basic_functions.Rmd

```diff
@@ -511,20 +511,6 @@ used in the   data, transformed data, or generated quantities blocks.*
 `R` **`fabs`**`(T x)`<br>\newline
 absolute value of x
 
-<!-- real; abs; (real x); -->
```

Comment:
Did this get moved to the deprecated section?  We keep bouncing back and forth about whether to sort out abs() or insist on fabs().

Reply:
added to deprecated section.